easybuild/easyconfigs/c/CVXOPT/CVXOPT-1.1.9-intel-2017a-Python-2.7.13.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = "PythonPackage"
+
+name = 'CVXOPT'
+version = '1.1.9'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://cvxopt.org'
+description = """CVXOPT is a free software package for convex optimization based on the Python programming language.
+ Its main purpose is to make the development of software for convex optimization applications straightforward by
+ building on Python's extensive standard library and on the strengths of Python as a high-level programming language.
+"""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/cvxopt/cvxopt/archive/']
+sources = ['%(version)s.tar.gz']
+checksums = ['09997fec809179c9bb9fe8cdd202ad6ecb675f890658219022f492e0797122ee']
+
+patches = ['%(name)s-%(version)s-fix-setup-py.patch']
```

Comment:
move `patches` above `checksums`, and also include SHA256 checksum for the patch file